sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_client_async.py

```diff
@@ -674,23 +607,18 @@ async def receive_message_batch_async(self, **kwargs):
         available rather than waiting to achieve a specific batch size, and therefore the
         number of messages returned per call will vary up to the maximum allowed.
 
-        If the receive client is configured with `auto_complete=True` then the messages received
-        in the batch returned by this function will already be settled. Alternatively, if
-        `auto_complete=False`, then each message will need to be explicitly settled before
-        it expires and is released.
-
-        :param max_batch_size: The maximum number of messages that can be returned in
+        :keyword max_batch_size: The maximum number of messages that can be returned in
          one call. This value cannot be larger than the prefetch value, and if not specified,
          the prefetch value will be used.
-        :type max_batch_size: int
-        :param on_message_received: A callback to process messages as they arrive from the
-         service. It takes a single argument, a ~uamqp.message.Message object.
-        :type on_message_received: callable[~uamqp.message.Message]
-        :param timeout: Timeout in seconds for which to wait to receive any messages.
+        :paramtype max_batch_size: int
+        :keyword on_message_received: A callback to process messages as they arrive from the
+         service. It takes a single argument, a ~pyamqp.message.Message object.
+        :paramtype on_message_received: callable[~pyamqp.message.Message]
+        :keyword timeout: Timeout in seconds for which to wait to receive any messages.
```

Comment:
Do we have a test for `timeout` ?

Reply:
Can make an issue to add with negative testing

Reply:
#24846